README.md

```diff
@@ -21,7 +21,7 @@ https://canonical-craft-store.readthedocs-hosted.com.
 To run all tests in the suite run:
```

Comment:
Above _Running tests_, how about adding:

```md
## Set up requirements

To install the necessary build tools, first run:

```bash
make setup
```.

```

Reply:
I went for "Set up a development environment" since I don't want to confuse "requirements" above with "adding new requirements" below. Whaddayathink?

Reply:
Bene!